sdk/core/azure-core/inc/http/policy.hpp

```diff
@@ -72,26 +74,31 @@ namespace Azure { namespace Core { namespace Http {
 
   struct RetryOptions
   {
-    int16_t MaxRetries = 5;
-    int32_t RetryDelayMsec = 500;
+    int32_t MaxRetries = 3;
```

Comment:
The conclusion of choice of integer types on Jun. 23 meeting is:
* `int64_t` for file length
* `size_t` or `ptrdiff_t` for array index
* `int64_t` or `int32_t` for integers that are transmitted/received over the wire

We never say we can't use `int` anymore, and `MaxRetries` isn't going to be sent over the wire. So why not just use `int`?

Reply:
You are right. I will change it, I actually like `int`s. But I'll double check with my team, in case I am missing something.

Reply:
Done.